src/components/Plans/Checkout/Checkout.js

```diff
@@ -160,13 +161,12 @@ const Checkout = InjectAppServices(({ dependencies: { dopplerBillingUserApiClien
             <div className="col-lg-12 col-md-6 col-sm-12 m-b-24"></div>
             <div className="col-sm-12 m-b-24">
               <hr className="dp-h-divider" />
-              <button
-                type="button"
+              <Link
```

Comment:
Will it work fine if you come here from another domain?

Reply:
Lo discutia con @jhoffman-ms  ayer, la unica forma de llegar es desde la calculadora de planes

Reply:
Mmmm, en ese caso, por que no directamente ir a la URL correspondiente en lugar de hacer un parche buscando en la historia?

Creo que sería más intuitivo y menos error-prone.

Reply:
no encontré una manera con react router, aunque también me serviría ir a la URL anterior pero con una recarga de la página, sabes de algo?

Reply:
Seguimos esta conversación en una llamada, pongo link al chat: https://makingsense.slack.com/archives/CRXR62TC2/p1655296333381309